### PR TITLE
docs(README,images): Pipeline architecture and diagrams / dataflow

### PR DESCRIPTION
**Pull request by MilagrosMarin**
This PR includes updates addressing [issue #77 in the aeon-docs repo](https://github.com/SainsburyWellcomeCentre/aeon_docs/issues/77) - from the list in [issue#60](https://github.com/SainsburyWellcomeCentre/aeon_docs/issues/60).

### CODE REVIEW

**Comment by MilagrosMarin** [comment]
Hi @thinh, I’ve implemented the changes we discussed in the internal meeting. The PR is now ready for a second round of review. Thank you!

File: aeon/dj_pipeline/README.md [hunk 1]
```diff
@@ -1,93 +1,89 @@
...
-
-8. `FoodPatchEvent` - all events (e.g. pellet triggered, pellet delivered, etc.)
-from a particular `ExperimentFoodPatch`
+#### Position data
 
-9. `FoodPatchWheel` - wheel data (angle, intensity) from a particular `ExperimentFoodPatch`
++ `qc.CameraQC` - quality control procedure applied to each `ExperimentCamera` (e.g. missing frame, etc.)
 
-10. `WheelState` - wheel states (threshold, d1, delta) associated with a given `ExperimentFoodPatch`
++ `tracking.SLEAPTracking` - position tracking for object(s), from a particular `VideoSource` per chunk
```
Comment (ttngu207) [inline aeon/dj_pipeline/README.md:58]: This is good.
Please expand more on the 2 part tables
- `PoseIdentity` - identified Subject (i.e. Identity) and stores the name of the body part used as "anchor_part"
- `Part` - inferred x,y position over time for all body parts from SLEAP model

File: aeon/dj_pipeline/docs/datajoint_overview_data_stream_diagram.svg
Comment (ttngu207) [inline aeon/dj_pipeline/docs/datajoint_overview_data_stream_diagram.svg:1]: This is great!
Please remove all of the part tables (`Attribute`, `RemovalTime`)
Please add `Epoch` in here as well

**Comment by MilagrosMarin** [comment]
@ttngu207 thanks for your suggestions. I've just added them.

**Review by ttngu207 — APPROVED** [review]

**Pull request merged**